mkdocs.yml

```diff
@@ -457,7 +457,6 @@ nav:
         - 'Adding authentication methods': 'casa/developer/add-authn-methods.md'
     - User Guide: 'casa/user-guide.md'
 plugins:
-- include-markdown
```

Comment:
hmm ... We may have to keep this plugin. That plugin I think is required for reuse of the text content. [PR](https://github.com/JanssenProject/jans/pull/8122/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2) from @misba7 . 

If we need to keep it, then I think it should be added to requirements as well so that plugin gets installed.

Reply:
Correct but it should be added with the requirements and it should not break the docs while we fix it.